sbt-version-policy/src/main/scala/sbtversionpolicy/internal/DependencySchemes.scala

```diff
@@ -0,0 +1,78 @@
+package sbtversionpolicy.internal
+
+import coursier.version.{ModuleMatcher, ModuleMatchers, VersionCompatibility}
+import lmcoursier.CoursierConfiguration
+import lmcoursier.definitions.Reconciliation
+import sbt.internal.librarymanagement.mavenint.SbtPomExtraProperties
```

Comment:
It is probably a bad idea to use this API since it’s named “internal”, right? (@eed3si9n)

Reply:
Internal means that we don't guarantee binary compatibility. In this case it's not likely we'd drop `VERSION_SCHEME_KEY` but if you want to be careful you might as well copy-paste the val definition.

Reply:
Thank you! Simpler to just inline it then.